schema/object.go

```diff
@@ -30,10 +27,10 @@ func (v Object) Validate(value interface{}) (interface{}, error) {
 		return nil, errors.New("not an object")
 	}
 	dest, errs := v.Schema.Validate(nil, obj)
-	if len(errs) > 0 {
-		var errMap ErrorMap
-		errMap = errs
-		return nil, errMap
+	if err := ErrorMap(errs).Tidy(); err != nil {
```

Comment:
Per my comment above, I would find it easier to read if it was:

```go
if len(errs) > 0 {
    return nil, ErrorMap(errs)
}
```

Reply:
Fixed